### ISSUE TITLE
Processes: similar structure like selection of flow names

### ISSUE THREAD

**Issue by Ecoxl** [Ecoxl/celero#58]

**Comment by Xandalfius** [comment]
Consistency:
Under Flow you have to click "Add Flow" first, which makes the fillable form visible, which is not the case for the other Tabs.

For Flow the last box is called "Description", for product (and the rest) it is "Comments". Just something I noticed, maybe thats's intentional then it's perfectly fine.

For flows, the Edit & Delete Button are on the end, while for Processes, Delete is in front, clickable for every used flow, while Edit is at the back to edit the comment.
![image](https://user-images.githubusercontent.com/69843619/91288727-a142b880-e791-11ea-834d-ad23cbd91f6e.png)

**Comment by tcgumus** [comment]
Yes we need to change the structure because there is no space left.
We couldn't find any solution to that. If you have, lets try it :)
